device/daita_types.go

```diff
@@ -0,0 +1,31 @@
+package device
+
+type EventType uint32
+
+const (
+	// NOTE: must be kept in sync with cmaybenot.h
+	NonpaddingSent     = EventType(0)
+	NonpaddingReceived = EventType(1)
+	PaddingSent        = EventType(2)
+	PaddingReceived    = EventType(3)
+)
+
+type Daita interface {
+	Disable()
```

Comment:
Consider if this would more suitably be named `Close()`. Disable implies that you're merely suspending processing, whereas "close" generally is understood to tear down an instance in golang.